### ISSUE TITLE
Lyrics do not appear in dribbblish

### ISSUE THREAD

**Issue by shreyashravi** [morpheusthewhite/spicetify-themes#170]
<img width="1183" alt="image" src="https://user-images.githubusercontent.com/54357427/93344319-29772380-f84f-11ea-954d-f5aab5aefe5d.png">

I'm on a mac. Been using this for a while, only noticed this today (didn't try to view lyrics before). I thought it was my spotify version so I reinstalled spotify, it works fine on basic spotify, but as soon as I apply dribbblish theme, I can't pull lyrics. Would greatly appreciate if this could be fixed.

**Comment by sortedcord** [comment]
Its not the theme that had done it, its spotify that has disabled the themes

**Comment by khanhas** [comment]
Might be some error log from console would help?
I'm sure tho, majority of Spotify users don't have access to Lyrics, myself included. I don't know how it works on yours and I don't want to debug the problem that I don't have a way to reproduce.

**Comment by shreyashravi** [comment]
Pardon my ignorance but how would I go around creating a spotify error log? Would the absence of lyrics be recorded on it? I can try digging in the source code myself but I doubt I ll get too far.

**Comment by khanhas** [comment]
1. Run `spicetify enable-devtools` in terminal
2. Spotify will be restarted
3. Hit <kbd>Ctrl + Shift + I</kbd> to open DevTools window
4. Navigate to tab Console
5. Open lyric app on Spotify window
6. Check Console window to see if there is any error.

**Comment by shreyashravi** [comment]
![Spotify lyrics](https://user-images.githubusercontent.com/54357427/94497940-5dbaee80-0216-11eb-8fd1-12f475caab0b.PNG)

The devtools window doesn't open on mac, so got it done on my windows instead. This is what I get. Although a point to note would be that I had these 3 errors as soon as I opened the devtool window. Pressing on the lyrics icon did not give lyrics, and also did not generate any new errors. I guess it's the second error listed?

**Comment by khanhas** [comment]
These "warning"s does not do anything, they are just devtool looking for source code map for debugging feature. Don't worry about them.
If there is no error, then sorry, I have no idea how to fix.

**Comment by wl03355609** [comment]
> These "warning"s does not do anything, they are just devtool looking for source code map for debugging feature. Don't worry about them.
> If there is no error, then sorry, I have no idea how to fix.
<img width="744" alt="截圖 2020-10-16 下午10 26 36" src="https://user-images.githubusercontent.com/57562404/96270851-bc49e000-0ffe-11eb-8730-2c47b9c18e1c.png">
maybe this?

**Comment by aaronleegit** [comment]
Anything that can contribute to this issue?
I have this issue as well, had to revert back to original spotify.

**Comment by AvinashReddy3108** [comment]
![Screenshot_20210416_205758](https://user-images.githubusercontent.com/27774996/115047821-6b8d4400-9ec8-11eb-9213-cb17540bfed9.png)

I'm running Spotify + Spicetify + [spotify-adblock-linux (forked)](https://gitlab.com/AvinashReddy3108/spotify-adblock-linux) on Arch linux, works fine for me now.
```bash
[avinash@rustbucket ~]$ spotify --version
Spotify version 1.1.56.595.g2d2da0de, Copyright (c) 2021, Spotify Ltd
```
I used to face this issue on an older version of Spotify tho (can't confirm it was because of Spicetify, coz never used it vanilla anyway)

**Comment by khanhas** [comment]
This problem was fixed in v1.2.1
Just upgrade spicetify.
